src/clr-icons/all.umd.ts

```diff
@@ -9,7 +9,4 @@ import { ApiModule } from './api';
 
 /* tslint:disable:variable-name */
 ApiModule.instance.init();
```

Comment:
What I meant is, you can remove this `ApiModule.instance.init();` now, since it's included in the next line.
So `init()` can become truly private if you want it to.